contributing-guides/style-guide.md

```diff
@@ -152,6 +152,7 @@ Keep the following guidelines in mind when choosing placeholders:
   and use the `{{/dev/sdXY}}` placeholder for *block devices* instead of `/dev/sda1`.
 - If a command can take a variable number of arguments, use an ellipsis: `{{arg1 arg2 ...}}`.
   If one of the multiple options is possible, write it as `{{either|or}}`.
+  If there are more than 5 alternatives use `|...` after the 5-th alternative.
```

Comment:
```suggestion
  If there are many alternatives, use `|...` for ellipsis.
```

Reply:
How about adding an article `the` before ellipsis. E.g 
> If there are many alternatives, use `|...` for the ellipsis

Reply:
Updated

Reply:
I've changed it to:

> If there are more than 5 alternatives, use `|...` for the ellipsis.